apps/wing-console/console/ui/src/features/map-view.tsx

```diff
@@ -253,9 +274,15 @@ const ConstructNode: FunctionComponent<PropsWithChildren<ConstructNodeProps>> =
                 "bg-white dark:bg-slate-700",
                 highlight && "bg-sky-50 dark:bg-sky-900",
                 "border",
-                "outline outline-0 outline-sky-200/50 dark:outline-sky-500/50",
-                !highlight && "border-slate-200 dark:border-slate-800",
-                highlight && "outline-4 border-sky-400 dark:border-sky-500",
+                "outline outline-0",
+                !highlight &&
+                  !hasError &&
+                  "border-slate-200 dark:border-slate-800",
+                highlight && !hasError && "border-sky-400 dark:border-sky-500",
+                highlight && "outline-4",
+                !hasError && "outline-sky-200/50 dark:outline-sky-500/50",
+                hasError &&
+                  "border-red-500 dark:border-red-800 outline-red-200/50 dark:outline-red-500/50",
```

Comment:
The contrast is not good for nodes with errors in dark mode.
![image](https://github.com/winglang/wing/assets/5547636/77d5b6e9-a066-4645-b8f6-f095f5812d08)

Reply:
It's brighter in dark mode now.